File: 58.length-of-last-word.java
/*
 * @lc app=leetcode id=58 lang=java
 *
 * [58] Length of Last Word
 */

// @lc code=start
class Solution {
    public int lengthOfLastWord(String s) {
        s = s.strip();
        String[] s_split = s.split(" ");
        return s_split[s_split.length-1].length();
    }
}
// @lc code=end

File: 58.length-of-last-word.py
#
# @lc app=leetcode id=58 lang=python3
#
# [58] Length of Last Word
#

# @lc code=start
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        is_continue = False
        count = 0
        for c in s:
            if c != " ":
                if is_continue:
                    count += 1
                else:
                    is_continue = True
                    count = 1
            else:
                is_continue = False

        return count


# @lc code=end
